### PR TITLE
feat(faucet): add faucet service

### PR DESCRIPTION
**Pull request by holic**
This is the last thing left in services package before we can 🔥

### CODE REVIEW

**Review by alvrs — DISMISSED** [review]

**Comment by alvrs** [comment]
amazing!

**Comment by alvrs** [comment]
(let's add a changeset with minimal usage instructions before merging)

File: examples/minimal/packages/client-react/src/mud/setupNetwork.ts [hunk 1]
```diff
@@ -44,28 +45,20 @@ export async function setupNetwork() {
...
-      if (lowBalance) {
-        console.info("[Dev Faucet]: Balance is low, dripping funds to player");
-        // Double drip
-        await faucet.dripDev({ address });
-        await faucet.dripDev({ address });
-      }
+    const drip = async () => {
+      console.log("dripping");
+      const tx = await faucet.drip.mutate({ address: burnerAccount.address });
+      console.log("got drip", tx);
```
Comment (alvrs) [inline examples/minimal/packages/client-react/src/mud/setupNetwork.ts:55]: should we still only do this if the balance is below a certain threshold?

**Review by alvrs — APPROVED** [review]

File: examples/minimal/packages/client-react/src/mud/setupNetwork.ts (hunk 1 above)
Comment (holic) [reply examples/minimal/packages/client-react/src/mud/setupNetwork.ts:55]: this is just the example app, but yes the templates should once we get this wired up to k8s

Comment (holic) [reply examples/minimal/packages/client-react/src/mud/setupNetwork.ts:55]: or better yet, consider putting the threshold in the faucet itself? so folks don't try to load up

**Pull request merged**